Javascript/README.md

```diff
@@ -49,6 +49,23 @@ if (time < 60) {
 }
 ```
 
+There is an objective reason for this. If you write the following code
+
+```javascript
+function myFunction()
+{
+	return
+	{
+		property1: 'value1',
+		property2: 'value2'
+	};
+}
+```
+
+`return\n` will be interpreted as `return;\n` and your function will be broken. Thus, for consistency matter, we want
+any curly braces to be on the same line as whatever they open.
```

Comment:
Exactly, that was the intention. Yet, that's not how it's interpreted. This is an example of how things can go wrong, and why opening curly braces are kept on the same line as what precedes it.